packages/components/src/components/dropdown-list/dropdown-list.jsx

```diff
@@ -64,7 +64,7 @@ const ListItems = React.forwardRef((props, ref) => {
                                 item_idx++;
                                 return (
                                     <ListItem
-                                        key={item_idx}
+                                        key={`item-${item_idx}`}
```

Comment:
I used the index here, because I had no other option. It's a bad practise, but not that bad if the elements in the data structure you're dealing with are not bound to change e.g. when they are not being sorted, rearranged or deleted.

Please help me check if there is a unique value I can use instead of this.

Thank you